README.md

```diff
@@ -200,8 +200,8 @@ In order for the Datadog Agent Spark integration to be able to collect metrics i
 mode the Spark and Spark History Server packages have to be configured according to the D2IQ security recommendations
 for this mode of operation.
 
-The following instructions are provided as a reference only. If you have already installed Spark and History Server you
-may skip this section.
+> :warning: The following instructions are provided as a reference only. If you have already installed Spark and History
+> Server you may skip this section.
```

Comment:
Right after you mention that one needs to confirm that HDFS is running with a service account, that there are permissions needed for the client and that there is a bare minimum list of permissions for spark. I don't know if one can install HDFS without all of these requirements, but if not, maybe not worth saying "skip the section". Could also add an anchor for people to skip to where you think they can.

Reply:
Great points, thank you 👍 HDFS and the client app are prerequisites for Spark History server so I should probably clarify that, will add a skip anchor as well.